lib/biokbase/Transform/script_utils.py

```diff
@@ -136,7 +136,7 @@ def parse_docs(docstring=None):
     remainder = script_details["Args"]
     argument_values = list()
     for k in reversed(argument_keys):
-        remainder, value = remainder.split(k)        
+        remainder, _, value = remainder.rpartition(k)       
```

Comment:
Is this feature absolutely necessary for rest of the code? If not I would try to keep this change in separate PR. It will be better in a way you don't create unnecessary bindings and you let the owner of repo to decide independently.